java/sage/api/FavoriteAPI.java

```diff
@@ -113,6 +113,21 @@ public Object runSafely(Catbert.FastStack stack) throws Exception{
       public Object runSafely(Catbert.FastStack stack) throws Exception{
         return Boolean.valueOf(((Agent) stack.pop()).testAgentFlag(Agent.DELETE_AFTER_CONVERT_FLAG));
       }});
+    rft.put(new PredefinedJEPFunction("Favorite", "IsEnabled", 1, new String[] { "Favorite" })
```

Comment:
Please use a more descriptive name than IsEnabled. The API call names are global...and while the other Favorite APIs could be more specific...'Enabled' is just too generic. Can you make it IsFavoriteEnabled?